supplier/tests/test_supplier_sign_up.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from django.template.loader import get_template
+from supplier.models import Supplier
+
+
+@pytest.mark.django_db()
+class TestSignupPage:
+    def test_signup_template(self):
+        get_template('supplier/supplier_sign_up.html')
```

Comment:
what is being tested here?

Reply:
Its outdated because it is fixed but bassicaly it was testing that there are a templated for signup,i will be doing get request